src/tests/utils/constants.js

```diff
@@ -0,0 +1,32 @@
+const testResults = [
+  {
+    id: 1,
+    revision: 'coconut',
+    author: 'johncleese@python.com',
+    push_timestamp: 42,
+    repository_id: 4,
+    revisions: [
+      {
+        revision: 'coconut',
+        author: 'johncleese@python.com',
+        comments: "you've got no arms left!",
+      },
+    ],
+  },
+  {
+    id: 2,
+    revision: 'spam',
+    author: 'ericidle@python.com',
+    push_timestamp: 42,
+    repository_id: 3,
+    revisions: [
+      {
+        revision: 'spam',
+        author: 'ericidle@python.com',
+        comments: "it's just a flesh wound",
+      },
+    ],
+  },
+];
+
+export default testResults;
```

Comment:
I believe there's a lint rule that throws an error "prefer default exports" if a file contains only one export. This might be another ESLint vs Prettier issue